robotpy_ext/control/toggle.py

```diff
@@ -1,61 +1,62 @@
 from _functools import partial
-from wpilib.timer import Timer
+
+import wpilib
 
 
 class Toggle:
-    """Utility class for button toggle
+    """Utility class for joystick button toggle
 
-        Usage:
+        Usage::
 
-        foo = Toggle(joystick, 3)
+            foo = Toggle(joystick, 3)
 
-        if foo:
-            toggleFunction()
+            if foo:
+                toggleFunction()
 
-        if foo.on:
-            onToggle()
+            if foo.on:
+                onToggle()
 
-        if foo.off:
-            offToggle()
+            if foo.off:
+                offToggle()
     """
     class _SteadyDebounce:
-        '''
+        """
             Similar to ButtonDebouncer, but the output stays steady for
             the given periodic_filter. E.g, if you set the period to 2
             and press the button, the value will return true for 2 seconds.
 
             Steady debounce will return true for the given period, allowing it to be
             used with Toggle
-        '''
+        """
 
         def __init__(self, joystick, button, period=0.5):
-            '''
+            """
             :param joystick:  Joystick object
             :type  joystick:  :class:`wpilib.Joystick`
```

Comment:
Whilst you're changing code around here, I think it would be a good idea to move these to type hints. I suspect this actually won't link to Joystick like this currently anyway.

Reply:
I suspect you're right